docs/src/languages/ruby.md

```diff
@@ -126,3 +126,59 @@ end
 <%= link_to "Hello", "/hello", class: "pl-2 <completion here>" %>
 <a href="/hello" class="pl-2 <completion here>">Hello</a>
 ```
+
+## Running tests
+
+To run tests in your Ruby project, you can set up custom tasks in your local `.zed/tasks.json` configuration file. These tasks can be defined to work with different test frameworks like Minitest, RSpec, quickdraw, and tldr. Below are some examples of how to set up these tasks to run your tests from within your editor.
+
+### Minitest
+
+```json
+[
+  {
+    "label": "test $ZED_RELATIVE_FILE:$ZED_ROW",
+    "command": "./bin/rails",
+    "args": ["test", "\"$ZED_RELATIVE_FILE:$ZED_ROW\""],
+    "tags": ["ruby-test"]
+  }
+]
+```
+
+### RSpec
+
+```json
+[
+  {
+    "label": "test $ZED_RELATIVE_FILE:$ZED_ROW",
+    "command": "./bin/rspec",
+    "args": ["\"$ZED_RELATIVE_FILE:$ZED_ROW\""],
+    "tags": ["ruby-test"]
+  }
+]
+```
+
+### quickdraw
+
+```json
+[
+  {
+    "label": "test $ZED_RELATIVE_FILE:$ZED_ROW",
+    "command": "./bin/qt",
```

Comment:
I don't think this is right unless there’s some separate process where binstubs are generated. Quickdraw is usually run with `bundle exec qt {file}`.

Reply:
@joeldrapper I used a `binstub` here because the `command` requires an absolute path. But I am not sure if this is still a thing. I will double-check that.

Reply:
Heeey, it actually works!

![CleanShot 2024-08-23 at 16 13 08@2x](https://github.com/user-attachments/assets/dba5fb88-b603-4c48-84f4-d30916d14a4a)


I will update example in this PR to `bundle exec` then. Thank you, @joeldrapper!